misc/acrn-config/launch_config/com.py

```diff
@@ -493,7 +497,8 @@ def dm_arg_set(names, sel, dm, vmid, config):
                 launch_cfg_lib.RE_CONSOLE_MAP['virtio-console(hvc0)']), file=config)
 
     if uos_type in ("CLEARLINUX", "ANDROID", "ALIOS"):
-        print("   -s {},virtio-net,$tap \\".format(launch_cfg_lib.virtual_dev_slot("virtio-net")), file=config)
+        if board_name in ("apl-mrb", "apl-up2"):
```

Comment:
@liuwei987  could you explain why apl-mrb, apl-up2 need this special handling, while others not?

Reply:
they are from the sample launch script, i think this could be treat as WA in acrn-config tools.
This will be fixed in the future.